.travis.yml

```diff
@@ -24,8 +24,18 @@ cache:
   - ${HOME}/.m2
 
 script:
-- ./gradlew plugin:build
+- ./gradlew plugin:assemble
 - ./gradlew plugin:publishToMavenLocal
 - ./gradlew generateSwagger
 - ./gradlew assembleDebug
 - ./gradlew check
+
+deploy:
+  provider: releases
+  api_key:
+    secure: Lw7e0w3MnEoBYInwlMyImQhHxTAUA46nnBrU66sLaawN9TJ8P8l/vOVpH7w2xYedVrPm57l+SnvrwzAqC/S8UBlgNNJBIE+3pQBwVGmsoiJ3uVvxOV+hIyhiXLQSQVCE71rSrCMX1aBysDoqLQW8UD29iG6a42LA4Moc4QzUNW1MBwEhixgVxpY7xFxzKBkm96WJDbe9MC6XyJEAJPgFO4Hj19shDFbZH71ofXc/RaBSv1OpX4L5CJUyfbzIBEtVvkVqfifqjlIUPrzAxxqIer2mIskCyzxcHLtlD/WrRrXB7MwBKMQFWDAGz4su2Dgs+k23DNZtbLBdf9GSrHaWeQGirJ7gJz25Dnsm7lVqzSJ4yVMrm3GBeBGCwmFwGn+Ff4x72XiSAtCC/ihAlAKMmUOo7otcN+ZW8OdZn4oiRemjvVy2y0UxdRQCiuaLV7m8jx3sovZjTXE2i5XeVPi3IW8T4srWkuDIYB4G88+iQDlkuNJSykHbgeCmUw8B5h5L65A9xxGVxswd8dKc1RV8F4AzTnQ6gSdtuQLdmmZpXEX10FmYMwPQ06vWgcU9wvDov+g0zI7RV3FAJKr4ESVU9LprpQZ9kllPvRqfbQ+rj6WJsOevT9dbTLIDoYRqdF7qzovBYWm8olIZPy7SOp4gwkbdSK8t/4U7MdwUnelmjc4=
+  file_glob: true
+  file: plugin/build/libs/*
+  on:
+    repo: Yelp/swagger-gradle-codegen
+    branch: prepare-release-1.1.0
```

Comment:
I would expect `tags: true` instead of `branch: ...`

Reply:
Side note: I would suggest to talk to #security (cc: @piax93) to have a machine-user to perform this type of releases

Reply:
Yup I just reverted the last commit.